src/components/matched/Matched.tsx

```diff
@@ -3,13 +3,18 @@ import { Header } from '@/components/header/Header'
 import { Footer } from '@/components/footer/Footer'
 import classes from '@/components/matched/style.css'
 import { UserInfoType } from '@/utils/types'
+import { useNavigate } from 'react-router-dom'
 
 //Recoil-on
 import { useRecoilValue } from 'recoil'
 import { userInfoState } from '@/atoms/userInfoAtom'
 
 export const Matched = () => {
   const [matchedUsers, setMatchedUsers] = useState<UserInfoType>({})
+  const navigate = useNavigate()
+  const transferToChat = (userId) => {
+    navigate('/chat', { state: { id: userId } })
```

Comment:
@masayuki654 @summerTakashi 
ここって、
```suggestion
    navigate(‘/chat’, { partnerId: userId })
```
ではエラーになるのでしょうか？
state:***という形式にする必要があるのかわかっていないです！

Reply:
@kan-mayoshi 
それだと通らないですね・・・
以下でどうでしょうか？
1ef878093f1b00982cee1b35d4258bac8f178540